oak-run/src/main/java/org/apache/jackrabbit/oak/run/Downloader.java

```diff
@@ -175,19 +172,18 @@ public ItemResponse call() throws Exception {
 
             Path destinationPath = Paths.get(item.destination);
             Files.createDirectories(destinationPath.getParent());
+
             long size = 0;
-            try (ReadableByteChannel byteChannel = Channels.newChannel(sourceUrl.getInputStream());
-                 FileChannel outputChannel = FileChannel.open(destinationPath, StandardOpenOption.CREATE, StandardOpenOption.WRITE)) {
-                ByteBuffer buffer = ByteBuffer.allocate(bufferSize);
+            try (InputStream inputStream = sourceUrl.getInputStream();
+                 FileOutputStream outputStream = new FileOutputStream(destinationPath.toFile())) {
```

Comment:
I wonder if using buffered output/input streams here would help in speeding up the download? Probably not. What was the rationale for not using buffered streams?

Reply:
I have not tested it. But I have compared this solution with the previous one based on channels and there is no performance degradation when checksum is not enabled.